app/views/topical_events/show.html.erb

```diff
@@ -3,43 +3,53 @@
 <% atom_discovery_link_tag atom_feed_url_for(@topical_event), t("topical_event.latest_activity", event: @topical_event.name) %>
 
 <%= content_tag_for(:div, @topical_event, class: "classification #{@topical_event.class.name.underscore}") do %>
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render "govuk_publishing_components/components/title", {
+          context: ("(Archived)" if @topical_event.archived?)&.html_safe,
+          title: h(@topical_event.name)
+        } %>
 
-  <header class="headings-block">
-    <div class="inner-block">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <%= render "govuk_publishing_components/components/title", {
-            title: "#{h(@topical_event.name)}#{' <span class="archived">(Archived)</span>' if @topical_event.archived?}".html_safe
-          } %>
-          <% if @topical_event.organisations.any? %>
+        <div class="page-description">
```

Comment:
Okay the `description` and `page-description` selectors were previously used to format govspeak content (in this case the remaining use was to format the lead paragraph copy).

Changes now include:
- removed both selectors (and associated styles) for `description` and `page-description`
- added a lead paragraph component which accepts text that has been processed by the `govspeak_to_html` and `strip_tags` helpers to remove any html (code sample below).

https://github.com/alphagov/whitehall/blob/cf0fc29bcd9d9c1da56cee82be7bc3eb7fda47ae/app/views/topical_events/show.html.erb#L14-L18

The reason for these changes is because some pages have heading markdown, in govspeak, where the lead paragraph copy should be e.g. https://www.gov.uk/government/topical-events/g7-uk-2021

```
##The UK hosted the G7 Summit as part of its 2021 G7 Presidency

The G7 Summit was held in Carbis Bay, Cornwall on 11-13 June 2021.
```

How does all this seems to you? It ties in in with some of the changes also carried out on this PR to separate out the page description in to a lead paragraph and a secondary description (detailed under _Anything else_)

Reply:
I think you've done the best you can with the wobbly data you've had to work with. Resolving 👍